engine/src/main/java/org/terasology/world/propagation/AbstractChunkView.java

```diff
@@ -1,85 +0,0 @@
-/*
- * Copyright 2013 MovingBlocks
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *      http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-package org.terasology.world.propagation;
-
-import org.terasology.math.JomlUtil;
-import org.terasology.math.geom.Vector3i;
-import org.terasology.world.block.Block;
-import org.terasology.world.internal.ChunkViewCore;
-
-/**
- * Intermediate abstract class for the propagater world view that handles common functionality.
- * <p>
- * Only provides a view for a single chunk
- *
- * @see AbstractFullWorldView
- */
-public abstract class AbstractChunkView implements PropagatorWorldView {
```

Comment:
Why remove this?

Reply:
This is not used anywhere in Omega. I moved this commit to a separate PR #4390 for clarity.